### DISCUSSION TITLE
FILOWA USB WiFi Adapter 1200 Mbps Realtek RTL8812BU Works

### DISCUSSION THREAD

**Discussion by pavoni** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#233] [General]
This one:

https://www.amazon.co.uk/gp/product/B078NSSM7W/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

Thanks for your work on this driver.

**Comment by chris1111** [comment]
Thanks I add to the working  list
